.gitignore

```diff
@@ -1,10 +1,10 @@
 .DS_Store
 node_modules
 .project
-dist/
```

Comment:
It looks like the updated Gruntfile no longer writes anything to `dist/`, so that's probably why.